aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,220 @@
+
+The main purpose of a page header is to identify a page and its context within an application. Therefore, the title and subtitle regions should receive screen priority when they come into contact with page-level actions.
+
+The region containing the title and subtitle should:
+
+- Flex to fill the available horizontal space on the page.
+- Have a minimum width of "medium" on `medium-xlarge` breakpoints and "small" on `xsmall-small` breakpoints to avoid excessive wrapping.
+
+Page-level actions should:
+
+- Have a width of "auto" by default.
+- Progressively collapse into an overflow menu as screen width becomes limited. This should follow guidance on [responsive behavior of actions](#responsive-behavior-of-actions).
+
+Resize your screen to see how the content regions adapt at various screen widths.
+
+<Example showResponsiveControls={[]} screenContainer plain width="100%">
+  <PageHeaderContentRegions background="background-front" />
+</Example>
+
+## Title
+
+A title is a required element of the page and should:
+
+- Always be an h1 with `size="small"`.
+- Uniquely and concisely identify the page. No two pages under the same parent should have the same name.
+- Follow title case capitalization (i.e., List of Clusters, My Account).
+- Wrap if too long for the available space. See [Responsive Behavior and Content Prioritization](#responsive-behavior-and-content-prioritization) for more guidance.
+- Be left-aligned with page content.
+
+## Subtitle
+
+A subtitle is an optional element that can provide additional contextual information about the contents of the page.
+
+A subtitle should:
+
+- Concisely elaborate what the page is on. This could be additional information such as ownership of an application, timestamps, or a name to represent by whom a piece of information was created.
+- Be located below the title and left-aligned with page content.
+- Wrap at the max-width for the paragraph's font-size in order to optimize the readability.
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{
+      type: 'do',
+      message:
+        'Include concise, high-priority metadata in the subtitle that aids page identification.',
+    }}
+    width="100%"
+    height={{ min: 'small' }}
+  >
+    <PageHeaderSubtitle />
+  </Example>
+  <Example
+    bestPractice={{
+      type: 'dont',
+      message: `Don't clutter the Page Header with all of the page metadata. Instead, include this in the page content.`,
+    }}
+    width="100%"
+    height={{ min: 'small' }}
+  >
+    <PageHeaderSubtitle bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Reverse Anchor
+
+A reverse Anchor is required on all child pages and should:
+
+- Be located above the title and left-aligned with page content.
+- Be labelled with the name of the parent page and accompanied by the "FormPrevious" icon.
+- Not be used on [top-level pages](#example-of-top-level-page) because they do not have a parent.
+
+### Example of Child Page
+
+<Example
```

Comment:
I know this is a preview in the preview container, but is there any way we can get the font-size proportions to show correctly on this preview? For instance, the reverse anchor looks way larger than the subtitle text
<img width="1493" alt="Screen Shot 2022-05-24 at 11 10 59 AM" src="https://user-images.githubusercontent.com/62117454/170103846-23267799-38e2-4d02-8c5c-5ff44744aff2.png">
.

Reply:
Yes, @halocline is going to help me with the anchor scaling.

Good callout about empty value. Fixed.

Reply:
Fixed anchor scaling thanks to Matt.